### PR TITLE
add films

### PR DESCRIPTION
**Pull request by BohdanBBG**

### CODE REVIEW

File: Lexiconner/Lexiconner.Domain/Entitites/FilmItemEntity.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    class FilmItemEntity: BaseEntity
+    {
+        public FilmItemEntity()
+        {
+            Tags = new List<string>();
+        }
+
+
+        public string UserId { get; set; }
+        public string FilmName { get; set; }
```
Comment (vadimberkut) [inline Lexiconner/Lexiconner.Domain/Entitites/FilmItemEntity.cs]: Just Title

File: Lexiconner/Lexiconner.Domain/Entitites/FilmItemEntity.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        public double Rating { get; set; }
```
Comment (vadimberkut) [inline Lexiconner/Lexiconner.Domain/Entitites/FilmItemEntity.cs]: Use decimal

File: Lexiconner/Lexiconner.Domain/Entitites/FilmItemEntity.cs [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Lexiconner.Domain.Entitites
+{
+    class FilmItemEntity: BaseEntity
```
Comment (vadimberkut) [inline Lexiconner/Lexiconner.Domain/Entitites/FilmItemEntity.cs]: Add ReleasedAt, Genres

Comment (vadimberkut) [inline Lexiconner/Lexiconner.Domain/Entitites/FilmItemEntity.cs]: Rename FilmItemEntity -> UserFilm

File: Lexiconner/Lexiconner.Domain/Entitites/FilmItemEntity.cs (hunk 1 above)
Comment (BohdanBBG) [reply Lexiconner/Lexiconner.Domain/Entitites/FilmItemEntity.cs]: ok

File: Lexiconner/Lexiconner.Application/ImportAndExport/FilmTxtImporter.cs [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+                    }
+
+                    result.Add(new FilmImportModel()
+                    {
+                        Title = title ?? null,
+                        Comment = comment ?? null,
+                        Rating = rating ?? null,
+                        WatchedAt = watchedAt ?? null,
+                        ReleasedAt = releasedAt ?? null,
+                        Genres = genres.ToList()?? null
```
Comment (vadimberkut) [inline Lexiconner/Lexiconner.Application/ImportAndExport/FilmTxtImporter.cs]: If `genres` is NULL `genres.ToList()` throws NullReferenceException. Try `genres?.ToList()`;

Comment (vadimberkut) [reply Lexiconner/Lexiconner.Application/ImportAndExport/FilmTxtImporter.cs]: Actually you do `List<string> genres = new List<string>();`, so `genres` can't be null

File: Lexiconner/Lexiconner.Domain/Config/MongoConfig.cs [hunk 5]
```diff
@@ -148,6 +149,15 @@ public static class MongoConfig
                 }
             },
 
+            new MongoCollectionConfig
+            {
+                CollectionType = typeof(UserFilm),
```
Comment (vadimberkut) [inline Lexiconner/Lexiconner.Domain/Config/MongoConfig.cs]: `UserFilm` -> `UserFilmEntity`

**Pull request merged**